pubsub/iterator.go

```diff
@@ -247,6 +247,8 @@ func (it *messageIterator) pullMessages(maxToPull int32) ([]*pb.ReceivedMessage,
 	switch {
 	case err == context.Canceled:
 		return nil, nil
+	case status.Code(err) == codes.Canceled:
```

Comment:
This will also now mask any rpcs that are canceled server side too, right? Is this okay?

Reply:
Just confirmed that server doesn't cancel any RPCs intentionally (they should always be initiated from client).

Reply:
Ack